sys/parsoid.js

```diff
@@ -621,9 +621,7 @@ PSP.transformRevision = function(hyper, req, from, to) {
             original: original
         };
         if (from === 'sections') {
-            body2.html = {
-                body: replaceSections(original, parseSections(req))
-            };
+            body2.html = replaceSections(original, parseSections(req));
```

Comment:
@gwicke Sorry, can you clarify what you want me to test? Section transforms already have tests ("sections2wt, replace", "sections2wt, append", etc.). They continue to pass because Parsoid accepts the both framed and unframed html. The change was really about consistency, not correctness, per se. You don't seem to have any other unit tests for ParsoidService, and it isn't even exported to the point where that can be attempted. What this patch changes is the value being sent by RESTBase to the Parsoid HTTP API, so I imagine what you want is to mock the Parsoid HTTP API and confirm the expectations about what's being posted. Please confirm.